addons/auth_totp_mail/models/auth_totp_device.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, models
+
+
+class AuthTotpDevice(models.Model):
+    _inherit = "auth_totp.device"
+
+    def unlink(self):
+        """ Notify users when trusted devices are removed from their account. """
+        removed_devices_by_user = self._classify_by_user()
+        for user, removed_devices in removed_devices_by_user.items():
+            user._notify_security_setting_update(
```

Comment:
Global remark: each change triggers its own email, is that intended ? I suppose we generally don't change that much touchy parameters at the same time, but I prefer to ask it this is intended.

Reply:
That was what I had in mind, so that's ok for me :)